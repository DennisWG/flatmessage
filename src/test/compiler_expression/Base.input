module Playground.Net.Base;

data Head
{
    uint8 code;
    uint16 size;
    optional uint32 crc;
}