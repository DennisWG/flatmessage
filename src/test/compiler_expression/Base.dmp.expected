enum: Color
data: Head
data: Vector2d
data: Vector3d
data: PlayerUpdateBody
message: PlayerUpdateDirection
message: PlayerUpdateVelocity
message: PlayerJumpStart
message: PlayerJumpEnd
